project-base/scripts/install.sh

```diff
@@ -17,7 +17,6 @@ echo "Start with specifying your operating system: \
 
     1) Linux
```

Comment:
Will it be straightforward? What do you thing about renaming this option to something like "Linux, Windows with WSL2"?

Reply:
I like this